### DISCUSSION TITLE
Question on the naming of the branches

### DISCUSSION THREAD

**Discussion by riioze** [SimVascular/SimVascular#1069] [Q&A]
Hi,
I'm currently using simvascular for ROM simulations and I'm quite confuse how the naming of the branches on the centerlines works (branchId and branchIdTmp) can anyone explain me the algorithm behind it ?
Thank you !

**Comment by ktbolt** [comment]
@riioze The centerlines geometry is explained in the `Centerlines Geometry` section of the [SimVascular ROM documentation](https://simvascular.github.io/documentation/rom_simulation.html#1d-solver).

As far as I know there is no documentation for the centerlines algorithm. Perhaps @mrp089 can explain a bit about how it works.
